lib/simplifyExpression/stepThrough.js

```diff
@@ -44,8 +44,8 @@ function stepThrough(node, debug=false) {
       logSteps(nodeStatus);
     }
     steps.push(removeUnnecessaryParensInStep(nodeStatus));
-    nodeStatus.reset();
-    nodeStatus = step(nodeStatus.newNode);
+    const nextNode = Status.resetChangeGroups(nodeStatus.newNode);
```

Comment:
merge your change to run tests+linter before committing ;D

you need to import `Status` here